### ISSUE TITLE
[Pattern] One good game is better than two great ones.

### ISSUE THREAD

**Issue by blinks** [blinks/pattern#15]
- [ ] Confidence, from 1 to 3 stars.
- [ ] Context (followed by `* * *`)
- [ ] Problem (in bold)
- [ ] Forces
- [ ] Solution (in bold, followed by `* * *`)
- [ ] "How and why" explanatory text.

https://www.gamasutra.com/view/news/114402/Analysis_Sid_Meiers_Key_Design_Lessons.php

**Comment by garykac** [comment]
Alternate link: http://www.designer-notes.com/?p=119
(Soren's blog)

**Comment by blinks** [comment]
Some of Sid's rules are difficult to actually apply to an existing design. I think this one might turn imperative as something more along the lines of pulling a smaller game out of a bigger one to make room for both.

**Comment by garykac** [comment]
I find this rule hard to parse. I keep trying to read it as "1 great is better than 2 good".

IIUC, he is trying to say: "taking 2 great ideas and cramming them into a single game will result in a bad game - it's better to focus on a single good game".

But then I ask: "why not focus on a single *great* game?" and "why not take both great ideas and make 2 great games"

**Comment by garykac** [comment]
> Some of Sid's rules are difficult to actually apply to an existing design. I think this one might turn imperative as something more along the lines of pulling a smaller game out of a bigger one to make room for both.

+1 to pulling smaller ideas out of the game. This is a common problem with people's first game - they want to cram all their great ideas into it.

This is similar to the advice given to new writers - don't cram all your great story ideas into your first book. it's OK to write multiple books.

**Comment by blinks** [comment]
Yup, 100%. If you figure out a better phrasing, change the title of this one. :)
